src/Components/Tasks/Tasks.tsx

```diff
@@ -76,24 +81,58 @@ const Tasks: React.FC = () => {
         getToastValue(
           "error",
           error?.response?.data?.message ||
-            "An error occurred. Please try again."
+          "An error occurred. Please try again."
         );
       });
   };
+  // **********get all Employee tasks**********pageSize:number, pageNumber:number*******
+  const getEmployeeTasksList = async () => {
+    await axios
+      .get(`${baseUrl}/Task`,
+        {
+          headers: requestHeaders,
+          // params:{
+          //   pageSize: 5,
+          //   pageNumber: pageNumber,
+          // }
+        })
+      .then((response) => {
+        // setTasks(response?.data?.data);
+        const todoTasks = response.data.data.filter((task: Itasks) => task.status == 'ToDo')
```

Comment:
you can improve this by abstracting a function that handle the filtration as follows:

```
    const filterTasksByStatus = (status) => data.data.filter((task) => task.status === status);

    const todoTasks = filterTasksByStatus('ToDo');
    const progressTasks = filterTasksByStatus('InProgress');
    const doneTasks = filterTasksByStatus('Done');
```